tekton/src/pipelines/taskdefs/dependencies/dro.yml.j2

```diff
@@ -0,0 +1,34 @@
+- name: dro
+  params:
+    {{ lookup('template', 'taskdefs/common/cli-params.yml.j2') | indent(4) }}
+    - name: devops_suite_name
+      value: dependencies-dro
+
+    - name: mas_instance_id
+      value: $(params.mas_instance_id)
+
+    - name: dro_action
+      value: "$(params.dro_action)"
+
+    - name: dro_contact_email
```

Comment:
Re-use uds params for username/firstname/lastname rather than create a whole load of new ones

Reply:
example:

```
    - name: dro_contact_email
      value: "$(params.uds_contact_email)"
```

This is the best point to convert the old names to the new, and it prevents the need to further complicate the inputs

Reply:
done suggested changes